warehouse/query-core/src/main/java/datawave/query/jexl/visitors/IteratorBuildingVisitor.java

```diff
@@ -1309,7 +1308,7 @@ public FunctionFilter(List<ASTFunctionNode> nodes) {
             if (nodes.size() > 1) {
                 ASTAndNode andNode = new ASTAndNode(ParserTreeConstants.JJTANDNODE);
                 children(script, andNode);
-                children(andNode, nodes.toArray(new JexlNode[nodes.size()]));
+                children(andNode, nodes.toArray(new JexlNode[0]));
```

Comment:
previously was the last node, now is the first?

Reply:
I remember delving into the baeldung analysis many years back.  Its a good article.